homeassistant/components/tuya/select.py

```diff
@@ -146,6 +147,13 @@
             device_class=DEVICE_CLASS_TUYA_MOTION_SENSITIVITY,
             entity_category=EntityCategory.CONFIG,
         ),
+        SelectEntityDescription(
```

Comment:
Please add a code comment to this one, as this is not a documented switch type in the API documentation.